protostar/cairo/cairo_bindings.py

```diff
@@ -60,24 +63,28 @@ def collect_tests(
             str(input_path),
             str(output_path) if output_path else None,
             [str(path) for path in cairo_path] if cairo_path else None,
-            RUNNER_BUILTINS_TITLE_CASE,
+            RUNNER_BUILTINS_TITLE_CASE + ["GasBuiltin"],
         )
-        return TestCollectorOutput(sierra_output=output[0], test_names=output[1])
+        return TestCollectorOutput(sierra_output=output[0], collected_tests=output[1])
 
 
 def compile_protostar_sierra_to_casm_from_path(
-    named_tests: list[str], input_path: Path, output_path: Optional[Path] = None
+    collected_tests: list[tuple[str, AvailableGas]],
```

Comment:
[nitpick]: `tuple[str, AvailableGas]` could be a small dataclass or a type

Reply:
It is dataclassed later into TeseCase, we will extend it to dataclass when we include more config information